.travis.yml

```diff
@@ -1,5 +1,5 @@
 before_install:
   - gem install bundler
 rvm:
-  - 1.9.3
   - 2.0.0
+  - 2.4.0
```

Comment:
Probably updating major version will be right. 1.9 support ended 2 years ago, do you know if anyone still using it? I haven't seen it for years

https://www.ruby-lang.org/en/news/2014/01/10/ruby-1-9-3-will-end-on-2015/